custom/icds/rules/immunization.py

```diff
@@ -152,7 +152,7 @@ def immunization_is_due(tasks_case, anchor_date, immunization_product, all_immun
     if product_schedule_flag:
         tasks_case_schedule_flag = tasks_case.get_case_property('schedule_flag')
         if (
-            not isinstance(tasks_case_schedule_flag, six.string_types) or
+            not isinstance(tasks_case_schedule_flag, (six.text_type, bytes)) or
```

Comment:
Is it even possible for `tasks_case_schedule_flag` to be `bytes` on Python 3? In other words, would it make sense to change this to `isinstance(tasks_case_schedule_flag, str)` after we switch to Python 3? (which means it might be reasonable to continue using `six.string_types` here)

Reply:
> Is it even possible for tasks_case_schedule_flag to be bytes on Python 3? 

I'm not sure, so I'm taking a very conservative approach to making changes with ```six.string_types```.